### PR TITLE
Add CI workflows for 19.lts.1+.

### PR DESCRIPTION
**Pull request by isarkis**

### CODE REVIEW

File: .github/actions/build/action.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          $GITHUB_WORKSPACE/cobalt/build/gyp_cobalt -v -C ${{ matrix.config }} ${{ matrix.platform }}
+          ninja -v -C ${GITHUB_WORKSPACE}/out/${{matrix.platform}}_${{matrix.config}} ${BUILD_TARGET}
+        else
+          $GITHUB_WORKSPACE/cobalt/build/gyp_cobalt -v -C ${{ matrix.config }} ${{matrix.bootloader_platform}}
+          if [[ "${{matrix.platform}}" =~ ^evergreen-x64 ]]; then
+            ninja -v -C ${GITHUB_WORKSPACE}/out/${{matrix.bootloader_platform}}_${{matrix.config}} elf_loader_sandbox
+            if [[ "${{matrix.config}}" == "devel" ]]; then
+              ninja -v -C ${GITHUB_WORKSPACE}/out/${{matrix.bootloader_platform}}_${{matrix.config}} loader_app_deploy elf_loader_sandbox_deploy crashpad_handler_deploy
+            fi
+          elif [[ "${{matrix.platform}}" =~ ^evergreen-arm-hardfp ]]; then
```
Comment (andrewsavage1) [inline .github/actions/build/action.yaml:31]: This is for all configs, not just devel?

File: .github/actions/on_host_test/action.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      uses: google-github-actions/setup-gcloud@v0.6.0
+    - name: Configure Environment
+      id: configure-environment
+      shell: bash
+      run: |
+        set -x
+        if [ "${{inputs.os}}" == 'linux' ]
+        then
+          echo "ARCHIVE_EXTENSION=tar.xz" >> $GITHUB_ENV
+        elif [ "${{inputs.os}}" == 'windows' ]
```
Comment (andrewsavage1) [inline .github/actions/on_host_test/action.yaml:20]: We don't have windows on this branch, do we need this?

File: .github/actions/upload_test_artifacts/action.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+name: Test Artifact Upload
+description: Uploads test archives to GCS and runs on-device tests.
+inputs:
+  type:
+    description: "Type of artifacts to upload (ondevice or onhost)"
+    required: true
+  os:
```
Comment (andrewsavage1) [inline .github/actions/upload_test_artifacts/action.yaml:7]: Isn't this always linux?

File: .github/actions/build/action.yaml (hunk 1 above)
Comment (isarkis) [reply .github/actions/build/action.yaml:31]: Looks like it, yes.

File: .github/actions/upload_test_artifacts/action.yaml (hunk 3 above)
Comment (isarkis) [reply .github/actions/upload_test_artifacts/action.yaml]: fixed.

File: .github/actions/on_host_test/action.yaml (hunk 2 above)
Comment (isarkis) [reply .github/actions/on_host_test/action.yaml]: removed.

**Comment by isarkis** [comment]
Removed windows support.

**Review by andrewsavage1 — APPROVED** [review]

**Pull request merged**